### PR TITLE
polish(pu): polish comments in env_wrappers.py and ding_env_wrapper.py

### PR DESCRIPTION
**Pull request by puyuan1996**
## Description
- polish comments in env_wrappers.py and ding_env_wrapper.py

## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

File: ding/envs/env/ding_env_wrapper.py [hunk 1]
```diff
@@ -59,7 +73,14 @@ def __init__(self, env: gym.Env = None, cfg: dict = None, seed_api: bool = True,
         self._replay_path = None
 
     # override
+
     def reset(self) -> None:
```
Comment (PaParaZz1) [inline ding/envs/env/ding_env_wrapper.py:77]: modify the corresponding type lint

File: ding/envs/env/ding_env_wrapper.py [hunk 2]
```diff
@@ -15,13 +15,27 @@
...
+        Overview:
+            Initialize the DingEnvWrapper. Either an environment instance or a config to create the environment
+            instance should be passed in:
+                - An environment instance: The `env` parameter must not be `None`, but should be the instance.
+                  It does not support subprocess environment manager. Thus, it is usually used in simple environments.
+                - A config to create an environment instance: The `cfg` parameter must contain `env_id`.
+        Arguments:
+            - env (:obj:`gym.Env`): An environment instance to be wrapped.
+            - cfg (:obj:`dict`): The configuration dictionary to create an environment instance.
+            - seed_api (:obj:`bool`): Whether to use seed API, default is True.
```
Comment (PaParaZz1) [inline ding/envs/env/ding_env_wrapper.py:37]: defaults to True

File: ding/envs/env/ding_env_wrapper.py [hunk 3]
```diff
@@ -15,13 +15,27 @@
...
+            - caller (:obj:`str`): The caller of this method, default is 'collector'.
```
Comment (PaParaZz1) [inline ding/envs/env/ding_env_wrapper.py:38]: add the total possible choices for this field

File: ding/envs/env_wrappers/env_wrappers.py [hunk 4]
```diff
@@ -85,41 +99,39 @@ def reset(self):
...
+    def __init__(self, env: gym.Env, skip: int = 4):
         """
         Overview:
-            Initialize ``self.`` See ``help(type(self))`` for accurate signature.
+            Initialize the MaxAndSkipWrapper.
         Arguments:
-            - env (:obj:`gym.Env`): the environment to wrap.
-            - skip (:obj:`int`): number of `skip`-th frame.
+            - env (:obj:`gym.Env`): The environment to wrap.
+            - skip (:obj:`int`): Number of `skip`-th frame. Defaults to 4.
```
Comment (PaParaZz1) [inline ding/envs/env_wrappers/env_wrappers.py:117]: ``skip``

File: ding/envs/env_wrappers/env_wrappers.py [hunk 5]
```diff
@@ -273,26 +288,37 @@ def reward(self, reward):
 class ActionRepeatWrapper(gym.Wrapper):
     """
     Overview:
-        Repeat the action to step with env.
-    Interface:
-        ``__init__``, ``step``
+        The ActionRepeatWrapper class is a gym wrapper that repeats the same action for a number of steps.
```
Comment (PaParaZz1) [inline ding/envs/env_wrappers/env_wrappers.py:291]: add more explanation for when to use this wrapper

File: ding/envs/env_wrappers/env_wrappers.py [hunk 6]
```diff
@@ -273,26 +288,37 @@ def reward(self, reward):
...
 
-    def step(self, action):
+    def step(self, action: Union[int, np.ndarray]) -> tuple:
+        """
+        Overview:
+            Take the given action and repeat it for a specified number of steps. The rewards are summed up.
+        Arguments:
+            - action (:obj:`Union[int, np.ndarray]`): the action to repeat.
+        Returns:
+            - obs (:obj:`np.ndarray`): the observation after repeating the action.
```
Comment (PaParaZz1) [inline ding/envs/env_wrappers/env_wrappers.py:317]: capitalize first letter

File: ding/envs/env_wrappers/env_wrappers.py [hunk 7]
```diff
@@ -306,31 +332,53 @@ def step(self, action):
 class DelayRewardWrapper(gym.Wrapper):
     """
     Overview:
-        Return cumulative reward at intervals; At other time, return reward of 0.
-    Interface:
-        ``__init__``, ``reset``, ``step``, ``new_shape``
+        The DelayRewardWrapper class is a gym wrapper that delays the reward. It cumulates the reward over a
```
Comment (PaParaZz1) [inline ding/envs/env_wrappers/env_wrappers.py:335]: which aims to simulate sparse reward environment

File: ding/envs/env_wrappers/env_wrappers.py [hunk 8]
```diff
@@ -347,27 +395,47 @@ def step(self, action):
...
+            Step the environment with the provided action, accumulate the returned reward, and add the total reward to
+            `info` if the episode is done.
+        Arguments:
+            - action (:obj:`Any`): The action to take in the environment.
+        Returns:
+            - obs (:obj:`np.ndarray`): The next observation from the environment.
+            - reward (:obj:`float`): The reward from taking the action.
+            - done (:obj:`bool`): Whether the episode is done.
+            - info (:obj:`Dict[str, Any]`): A dictionary of extra information, which includes 'eval_episode_return' if the
+              episode is done.
```
Comment (PaParaZz1) [inline ding/envs/env_wrappers/env_wrappers.py:437]: add `Examples` for this wrapper

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: ding/envs/env/ding_env_wrapper.py (hunk 3 above)
Comment (puyuan1996) [reply ding/envs/env/ding_env_wrapper.py:38]: polished

File: ding/envs/env/ding_env_wrapper.py (hunk 2 above)
Comment (puyuan1996) [reply ding/envs/env/ding_env_wrapper.py:37]: The original comment seems to be grammatically correct.

File: ding/envs/env_wrappers/env_wrappers.py (hunk 5 above)
Comment (puyuan1996) [reply ding/envs/env_wrappers/env_wrappers.py:291]: polished

File: ding/envs/env_wrappers/env_wrappers.py (hunk 6 above)
Comment (puyuan1996) [reply ding/envs/env_wrappers/env_wrappers.py:317]: fixed

File: ding/envs/env_wrappers/env_wrappers.py (hunk 7 above)
Comment (puyuan1996) [reply ding/envs/env_wrappers/env_wrappers.py:335]: polished

File: ding/envs/env_wrappers/env_wrappers.py (hunk 8 above)
Comment (puyuan1996) [reply ding/envs/env_wrappers/env_wrappers.py:437]: done

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**